frontend/src/services/objectService.ts

```diff
@@ -295,8 +301,7 @@ export default {
       );
     }
 
-    const fd = new FormData();
-    fd.append('file', object);
-    return comsAxios(axiosOptions).post(`${PATH}/${objectId}`, fd, config);
+    const fd = await object.arrayBuffer();
```

Comment:
I'm not completely certain what the await does, but does this end up forcing the browser to consume a large file payload (multiple gigs) and shove into memory if we end up awaiting? Just slightly concerned about potential adverse effects to browser memory usage on significantly large arrayBuffered files.

Reply:
After testing on some images, it doesn't seem like there's a synchronous lock on the browser while it uploads. But that isn't a full test on browser memory and possible adverse effects.